CMakeLists.txt

```diff
@@ -647,6 +646,8 @@ endif()
 
 if(LLGL_BUILD_RENDERER_OPENGL)
     # OpenGL Renderer
+    # set OpenGL preference
+    set(OpenGL_GL_PREFERENCE "GLVND")
```

Comment:
Actually this patch is totally useless if we read the CMake documentation.

```
If the GLVND OpenGL and GLX libraries are available, prefer them. This forces OPENGL_gl_LIBRARY to be empty. This is the default if components were requested (since components correspond to GLVND libraries).
```

This means that GLVND will anyway be chosen if all the components are found. Notice also the next line after find_package
```
if(OpenGL_FOUND)
```
that guarantess that
```
True, if the system has OpenGL and all components are found.
```

So, because of all the safe defaults taken in place automatically by CMake we can definitely skip this line.

Ref.: https://cmake.org/cmake/help/v3.16/module/FindOpenGL.html